.github/workflows/release-creation.yml

```diff
@@ -45,9 +45,9 @@ jobs:
     - name: Install Dependencies
       run: |
         # Install deps in caller repo
-        npm ci
+        npm install
```

Comment:
The only concern I have in this PR is this line. I do not expect any problem as `npm install` should still install all dependencies. The difference would be if a repo that is reusing this workflow has updates to `package.json` and _not_ `package-lock.json`. In that case, `npm ci` would notice the discrepancy and generate an error. Whereas `npm install` will install the dependencies as noted by `package.json`.